### PR TITLE
TRW-1429 - double clicking a table row with a link navigates to link

### PR DESCRIPTION
**Pull request by troyblank**
Adds the double click link for universal search rows.

In order to get double click link to work well with testing for our universal search table I had to separate  `<Table />` into two different components adding `<Row />`. This allows us to handle the click event more cleanly.

### CODE REVIEW

**Comment by mathom** [comment]
[100% (0.0%) vs master 100%](https://jenkins.topbox.io/job/Topbox/job/rci-components/job/PR-637/1/)

File: components/table/row.scss [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+.highlights {
+    @include breakpoint-below($size-tablet) {
+        margin-left: 0;
+    }
+
+    display: grid;
+    margin-bottom: $small-spacing;
+    grid-gap: 1rem;
+    grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
```
Comment (tVeratti) [inline components/table/row.scss]: Fancy. I like the use of grid here. I wasn't familiar with this way of setting the columns - just did a bit or reading on this. Noice.

File: test/components/table/row.spec.js [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+            row={row}
+            cellStyles={cellStyles}
+            link={link}
+          />
+        );
+        const linkItem = wrapper.find(`.${styles.row}`);
+
+        window.location.assign = sinon.spy();
+        linkItem.simulate('dblclick');
+        assert.isTrue(window.location.assign.calledWith(link));
```
Comment (tVeratti) [inline test/components/table/row.spec.js]: Thorough tests, nice.

File: components/table/row.scss (hunk 1 above)
Comment (troyblank) [reply components/table/row.scss]: Yea, the huge benefit is a lot less media queries.

**Pull request merged**